web/modules/custom/joinup_core/joinup_core.deploy.php

```diff
@@ -182,3 +185,43 @@ function joinup_core_deploy_0106802(array &$sandbox): string {
 
   return "Processed {$sandbox['progress']} out of {$sandbox['total']}";
 }
+
+/**
+ * Update the EIRA vocabulary.
+ */
+function joinup_core_deploy_0106803(array &$sandbox): void {
+  // Clean up the existing graph.
+  $sparql_connection = Database::getConnection('default', 'sparql_default');
+  $sparql_connection->query('WITH <http://eira_skos> DELETE { ?s ?p ?o } WHERE { ?s ?p ?o } ');
+
+  // Re import the file to update the terms.
+  $connection_options = $sparql_connection->getConnectionOptions();
+  $connect_string = "http://{$connection_options['host']}:{$connection_options['port']}/sparql-graph-crud";
+  $graph_store = new GraphStore($connect_string);
```

Comment:
Why we're doing this again and again? There's the `SparqlGraphStoreTrait::createGraphStore()` helper from a long time :)